core/src/main/java/org/elasticsearch/indices/analysis/AnalysisModule.java

```diff
@@ -262,6 +271,33 @@ public AnalysisRegistry getAnalysisRegistry() {
         return tokenFilters;
     }
 
+    static Map<String, PreBuiltTokenFilterSpec> setupPreBuiltTokenFilters(List<AnalysisPlugin> plugins) {
+        NamedRegistry<PreBuiltTokenFilterSpec> preBuiltTokenFilters = new NamedRegistry<>("pre built token_filter");
+
+        // Add filters available in lucene-core
+        preBuiltTokenFilters.register("lowercase", new PreBuiltTokenFilterSpec(true, CachingStrategy.LUCENE, (inputs, version) ->
```

Comment:
I debated moving `lowercase` out of core and just keeping standard but there isn't really a technical reason to do that so I decided not to for now.